### ISSUE TITLE
Provide SHAKE test vectors with fixed output length

### ISSUE THREAD

**Issue by jvdsn** [usnistgov/ACVP-Server#184]
Currently, the ACVP protocol requires a domain for the SHAKE `outputLen` value. As a result, it's impossible to request test vectors for which the output length is fixed, i.e. the domain only contains one value. When `min` and `max` are set to the same value, the ACVP server reports an error:
```
[
  {
    "acvVersion": "1.0"
  },
  {
    "error": "Validation error(s) on JSON payload.",
    "context": [
      "SHAKE-128-1.0: min must be less than max",
      "SHAKE-256-1.0: min must be less than max"
    ]
  }
]
```

Why is this restriction set on the output length?

**Comment by jvdsn** [comment]
Hi @livebe01 I'm seeing this was assigned to a milestone. Does this mean the second request I showed, i.e. with:
```
        "outputLen": [
          136
        ]
```
will be valid in a future update of the ACVP server? Do you have any idea when the update will be published?

**Comment by livebe01** [comment]
The fix for this is now on Prod in release [v1.1.0.25](https://github.com/usnistgov/ACVP-Server/releases/tag/v1.1.0.25).
